applications/StructuralMechanicsApplication/custom_utilities/structural_mechanics_element_utilities.h

```diff
@@ -21,8 +24,78 @@
 #include "includes/element.h"
 
 namespace Kratos {
+/**
+ * @namespace ElementUtilities
+ * @ingroup StructuralMechanicsApplication
+ * @brief This class includes several utilities necessaries for the computation of the different elements
+ * @author Philipp Bucher
+ * @author Vicente Mataix Ferrandiz
+ * @author Riccardo Rossi
```

Comment:
i have done nothing of this. i don't need to be included

Reply:
Yes, because I am refactoring code that was yours, that's why